Bump moondeck to v1.4.0

FrogTheFrog opened the pull request:
# MoonDeck

Changes:
- Plugin now uses a HTTPS protocol instead of a custom one (technical debt). Simplifies things a lot, but brings nothing new apart from that.
- Added an option to suspend the PC.

## Checklist:

### Developer Checklist

- [x] I am the original author or an authorized maintainer of this plugin.
- [x] I have abided by the licenses of the libraries I am utilizing, including attaching license notices where appropriate.

### Plugin Checklist

- [x] I have verified that my plugin works properly on the Stable and Beta update channels of SteamOS.
- [x] I have verified my plugin is unique or alternatively provides more/alternative functionality to a similair plugin already on the store.
- [ ] I am using a custom backend other than Python.
- [ ] I am using a tool or software from a 3rd party FOSS project that does not have it's depedencies [statically linked](https://en.wikipedia.org/wiki/Static_library).
- [ ] I am using a custom binary that has all of it's depedencies statically linked.

FrogTheFrog commented:
Changes since last build:
- Custom resolution now takes priority over automatic one since it detects native resolution mostly. Very useful for external displays.
- Multiple custom resolutions can be saved now per host.
- Custom resolution can now be toggled and selected in quick access menu.

nabel0 reviewed (approved):
Tested on steamos Stable, beta and preview.
Works perfect...

FrogTheFrog commented:
Change since last build (hopefully final):
- Removed early resolution change option as it is a leftover from NVidia times.
- Some QOL improvements here and there
